src/main/java/health/ere/ps/service/muster16/parser/rgxer/delegate/pattern/PractitionerPatterns.java

```diff
@@ -0,0 +1,12 @@
+package health.ere.ps.service.muster16.parser.rgxer.delegate.pattern;
+
+import java.util.regex.Pattern;
+
+public class PractitionerPatterns extends Patterns {
+    public final Pattern NAME_PREFIX = Pattern.compile("(Dr)\\.");
```

Comment:
Other possible prefixes: (med. prof.), can be a composite prefix like: "Dr. med."

Reply:
Ok, support of them added.